std/path.d

```diff
@@ -4005,11 +4005,14 @@ string expandTilde(string inputPath) nothrow
 }
 
 
-version(unittest) import std.process : environment;
+version(unittest)
```

Comment:
You can remove this now ...

Reply:
I was wondering about that construct. :-)
Didn't understand why the import was placed there.